skxray/fitting/xrf_model.py

```diff
@@ -1117,12 +1116,12 @@ def linear_spectrum_fitting(spectrum, params, elemental_lines=None,
         if np.sum(total_y[:, i]) == 0:
             continue
         result_dict.update({total_list[i]: total_y[:, i]})
-        area = 0
+        area = None
         if area_option:
             area = out[i]*element_area[total_list[i]]
```

Comment:
This function is also used in single pixel fitting, so I prefer to reduce unnecessary calculation.